lib/licensed/sources/gradle.rb

```diff
@@ -66,7 +66,7 @@ def executable
         return @executable if defined?(@executable)
 
         @executable = begin
-          gradlew = File.join(config.pwd, "gradlew")
```

Comment:
For gradle multi-projects builds, the `config.pwd` is often a subfolder and not the root of the project.
I think licensed should look for gradlew at the root since it is the official location for the gradle wrapper

What do you think about that ?